src/imp/span_context_imp.js

```diff
@@ -0,0 +1,40 @@
+export default class SpanContextImp {
+
+    // ---------------------------------------------------------------------- //
+    // OpenTracing Implementation
+    // ---------------------------------------------------------------------- //
+
+    setBaggageItem(key, value) {
+        this._baggage[key] = value;
+    }
+
+    getBaggageItem(key) {
+        return this._baggage[key];
+    }
+
+    // ---------------------------------------------------------------------- //
+    // LightStep Extensions
+    // ---------------------------------------------------------------------- //
+
+    // This is part of the formal OT API in Go; and will likely be supported
+    // across platforms.
+    //
+    // https://github.com/opentracing/opentracing.github.io/issues/103
+    foreachBaggageItem(f) {
+        for (let key in this._baggage) {
+            if (!f(key, this._baggage[key])) {
+                return;
+            }
```

Comment:
I was hoping to satisfy something like http://underscorejs.org/#each and something like http://underscorejs.org/#find with a single API.